samples/metaapp/src/main/res/layout-television/property_list_title.xml

```diff
@@ -12,19 +12,6 @@
     app:layout_constraintEnd_toEndOf="parent"
     tools:ignore="MissingDefaultResource">
 
-    <TextView
-        android:id="@+id/titleTextView"
-        android:textAppearance="@android:style/TextAppearance.Large"
-        android:gravity="center"
-        android:layout_width="wrap_content"
-        android:layout_height="80dp"
-        android:layout_marginEnd="24dp"
-        android:layout_centerVertical="true"
-        android:focusable="false"
-        android:layout_margin="6dp"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
     <ImageButton
         android:id="@+id/removeAllBtn"
```

Comment:
From the style point of view, it looks better switching the position of the 2 buttons:
+ btn first and cancel btn second